pkg/liqo-controller-manager/namespaceOffloading-controller/namespaceoffloading_lifecycle.go

```diff
@@ -49,7 +50,16 @@ func (r *NamespaceOffloadingReconciler) initialConfiguration(ctx context.Context
 	ctrlutils.AddFinalizer(noff, namespaceOffloadingControllerFinalizer)
 	// 2 - Add empty cluster selector if not specified by the user.
 	if noff.Spec.ClusterSelector.Size() == 0 {
-		noff.Spec.ClusterSelector = corev1.NodeSelector{NodeSelectorTerms: []corev1.NodeSelectorTerm{}}
+		noff.Spec.ClusterSelector = corev1.NodeSelector{NodeSelectorTerms: []corev1.NodeSelectorTerm{
```

Comment:
The selector is no more empty. Can you please update the comment and explain why was it required?